src/components/LandingPage/LandingPage.js

```diff
@@ -42,6 +43,7 @@ const LandingPage = (props) => {
                 <h4>Popular</h4>
                 <div className='row card-section'>
                     {dummy_json.map((course, idx) => {
```

Comment:
I guess I don't understand the point of having "dummy" json here -- can you clarify?

Reply:
Dummy_json is just a placeholder for the "Popular" cards on the landing page. It will be replaced with an actual set of "spotlighted" courses since there is no "popular" query in ES -  https://jira.deloitte.com/browse/ECC-551
Let me know if you have a different take on this.

Reply:
Got it -- fine as-is for now -- we can discuss enhancement ideas on the ticket.